src/components/dataset/overview/connection-to-forms.vue

```diff
@@ -174,22 +174,19 @@ export default {
 <i18n>
 {
   "de": {
-    "formsUpdateEntities": "Formular aktualisiert Objekte in dieser Objektliste | Formulare aktualisieren Objekte in dieser Objektliste"
+    "formsUpdateEntities": "{count} Formular, die Entitäten aktualisieren. | {count}Formulare, die Entitäten aktualisieren."
   },
   "es": {
-    "formsUpdateEntities": "El formulario actualiza Entidades en esta Lista de Entidades | Los formularios actualizan Entidades en esta Lista de Entidades | Los formularios actualizan Entidades en esta Lista de Entidades"
+    "formsUpdateEntities": "{count}Formulario que actualiza entidades. | {count}Formularios que actualizan entidades. | {count} Formularios que actualizan entidades."
```

Comment:
The translator forgot to put a space between `{count}` and "Formulario". I'll fix that in Transifex later (as well as other similar instances).